pkg/apis/core/types_seed.go

```diff
@@ -139,6 +141,39 @@ type SeedProvider struct {
 	Region string
 }
 
+// SeedSettings contains certain settings for this seed cluster.
+type SeedSettings struct {
+	// ExcessCapacityReservation controls the excess capacity reservation for shoot control planes in the seed.
+	ExcessCapacityReservation *SeedSettingExcessCapacityReservation
+	// Scheduling controls settings for scheduling decisions for the seed.
+	Scheduling *SeedSettingScheduling
+	// ShootDNS controls the shoot DNS settings for the seed.
+	ShootDNS *SeedSettingShootDNS
```

Comment:
Hm, not sure whether I get this one. We already have `.spec.dns` in the `Seed`. Now having `.spec.settings.dns` will be misleading imo. Hence, I explicitly called it `ShootDNS` because all the configuration related to the seed DNS is/will be part of `.spec.dns` already.

What do you mean by "for example the monitoring and logging in the garden namespace"?